content/applications/finance/payment_providers.rst

```diff
@@ -236,6 +236,9 @@ To capture the payment after it was authorized, go to the related sales order or
 on the :guilabel:`CAPTURE TRANSACTION` button. To release the funds, click on the :guilabel:`VOID
 TRANSACTION` button.
 
+If your provider supports the partial capture, you may choose the amount you want to capture, and
+tick whether the remaining amount has to be voided or wait for another partial capture.
```

Comment:
I'm not sure if that'd be clear for a customer. It is to me because I wrote the spec, but it sounds odd IMO.
@toaa-odoo Do you have a better alternative?

Reply:
If your provider supports partial capture, you can select to capture only a **partial amount** of the full amount. The **remaining (uncaptured) amount** can either be voided or be partially captured again at a later date.

Not sure how the "wait for another partial capture" part works, but this is my suggestion based on what I understood :)